docs/features/appgw-ssl-certificate.md

```diff
@@ -48,7 +48,16 @@ appgwName=""
 resgp=""
 vaultName=""
 location=""
-agicIdentityPrincipalId=""
+aksClusterName=""
+aksResourceGroupName=""
+appgwName=""
+
+# get the resource group name of the AKS cluster
```

Comment:
This approach will only work for the addon. Can you please add a comment here?

Reply:
If you intend the naming convention for the AGIC managed identity, this is true. Let me add a comment.